pillar/elife-public.sls

```diff
@@ -102,10 +92,6 @@ elife:
                 - javier
                 - hem
                 - erez
-            elife-xpub:
-                - javier
-                - hem
-                - erez
             profiles:
                 - giancarlo
```

Comment:
Guess this should be removed, as Giancarlo doesn't work here anymore.

Reply:
Can be addressed in separate PR.

Reply:
Yeah, no need to hold this PR up on that

Reply:
quick note, and I should probably add it to the yaml as a comment, but if you remove a key from a project it should be moved to the `denied` section. This will ensure that key is cleaned up.

in this case where the elife-xpub project is going away entirely, we remove both `allowed` and `denied` sections for the project